x/programs/rust/examples/nft/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "nft"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+wasmlanche_sdk = { version = "0.1.0", path = "../../wasmlanche_sdk", features = ["simulator"]}
```

Comment:
is `simulator` a `dev-dependency`?

Reply:
This feature is tracked in https://github.com/rust-lang/cargo/issues/7916. It's in nightly but not sure if it's in stable.